### ISSUE TITLE
TestDummyTask.test_dummy_task_computation is flaky

### ISSUE THREAD

**Issue by Wiezzel** [golemfactory/clay#2270]
This test failed twice for my PR, but when I re-run it for the third time it passed.
Link to the builds: https://circleci.com/gh/golemfactory/golem/tree/2234_golemcli_incomes
Error message:
```
=================================== FAILURES ===================================
__________________ TestDummyTask.test_dummy_task_computation ___________________

self = <tests.golem.task.dummy.test_dummy_task.TestDummyTask testMethod=test_dummy_task_computation>

    def test_dummy_task_computation(self):
        error_msg = runner.run_simulation(
            num_computing_nodes=2, num_subtasks=3)
>       self.assertIn(error_msg, [None, "Node exited with return code 0"])
E       AssertionError: 'Computation timed out' not found in [None, 'Node exited with return code 0']

tests/golem/task/dummy/test_dummy_task.py:14: AssertionError
```

**Comment by Krigpl** [comment]
Looks like it will be fixed by #2268

**Comment by maaktweluit** [comment]
I hope this is resolved with the linked PR, please re-open if not the case and i'll investigate
